src/orca-loader/Elements.res

```diff
@@ -27,7 +27,13 @@ let make = (
     let logger = logger->Option.getOr(OrcaLogger.defaultLoggerConfig)
     let savedPaymentElement = Dict.make()
     let localOptions = options->JSON.Decode.object->Option.getOr(Dict.make())
-    let endpoint = ApiEndpoint.getApiEndPoint(~publishableKey, ())
+    let apiEndPoint = ApiEndpoint.apiEndPoint.contents->Option.getOr("")
+    let endpoint = if apiEndPoint->String.length === 0 {
```

Comment:
I think this check of whether apiEndpoint is empty or not is already present inside getApiEndPoint